src/pages/homePage/organism/mainProductSection/molecules/productHead/atoms/Filter.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
```

Comment:
Atoms are prestational components made up of html attributes, only used for presentation.
This component is filtering please create such components as molecules.